tutorials/users/remote-processing/remote-processing.md

```diff
@@ -1,7 +1,199 @@
 # Remote Processing
 
-```{important}
-This tutorial is coming soon!
+Remote processing services allow users to run workflows on different nodes across the Marble network.
 
-Check back soon for more content.
+By distributing climate data across the nodes in the Marble network, users can choose to run their workflows on the same
+servers as the data. In other words, instead of downloading large amounts of data in order to run workflows on their own
+computers, users can send their workflows to the computer that hosts the data to run there. This reduces lengthy
+download times and eliminates the need to duplicate data unnecessarily.
+
+The [Weaver](https://pavics-weaver.readthedocs.io) service provides remote processing capabilities across the Marble 
+Network. Any node that enables the Weaver service will allow users to run workflows on that node from anywhere else in 
+the network.
+
+## How Weaver works
+
+Weaver is an execution management service (EMS) where users can send instructions in the form of workflows which are 
+then executed by different processes across the network. A workflow can contain a single process step or multiple.
+
+Processes can either be run in synchronous or asynchronous mode. 
+
+- A synchronous process will block until it has finished and then return the result of the process. 
+
+- An asynchronous process will immediately return a response containing a URL that can be used to check the status of 
+  the process and to retrieve the result once the process has finished. A user running an asynchronous process is 
```

Comment:
I'm not sure I follow this specific comment about responsibility. Since a synchronous process is blocking, isn't a user also responsible for checking back later to see if a synchronous process has finished? Is there a difference between the two cases?

Reply:
Yeah. In synchronous mode, the user will get the response back when the process has finished. In asynchronous mode, the user won't